examples/src/main/java/io/scalecube/examples/ClusterMetadataExample.java

```diff
@@ -47,6 +47,8 @@ public static void main(String[] args) throws Exception {
     if (joeMemberOptional.isPresent()) {
       alice.send(joeMemberOptional.get(), Message.fromData("Hello Joe"));
     }
+
+    TimeUnit.SECONDS.sleep(3);
```

Comment:
Seems it misses import - https://travis-ci.org/scalecube/scalecube/builds/168205838#L2098